apps/web/next.config.js

```diff
@@ -12,26 +14,65 @@ const transpileModules = [
   '@sindresorhus/transliterate', // Used by slugify.
   'escape-string-regexp', // Used by slugify.
 ]
-const withTM = require('next-transpile-modules')(transpileModules)
 
-const { API_URL = 'http://localhost:4444' } = process.env
+const withTM = require('next-transpile-modules')(transpileModules)
 const graphqlPath = '/api/graphql'
 
-module.exports = withTreat(
-  withTM(
-    withHealthcheckConfig({
-      cssModules: false,
-      serverRuntimeConfig: {
-        // Will only be available on the server side
-        // Requests made by the server are internal request made directly to the api hostname
-        graphqlUrl: API_URL,
-        graphqlEndpoint: graphqlPath,
-      },
-      publicRuntimeConfig: {
-        // Will be available on both server and client
-        graphqlUrl: '',
-        graphqlEndpoint: graphqlPath,
-      },
-    }),
+const {
+  API_URL = 'http://localhost:4444',
+  SENTRY_DSN,
+  SENTRY_AUTH_TOKEN,
+  NODE_ENV,
+} = process.env
+
+module.exports = withSourceMaps(
+  withTreat(
+    withTM(
+      withHealthcheckConfig({
+        webpack: (config, options) => {
+          if (!options.isServer) {
+            config.resolve.alias['@sentry/node'] = '@sentry/browser'
+          }
+
+          /*
+          const include = path.join(
```

Comment:
@eirikurn Two questions about this:

- When running ` yarn start web --configuration="production"` the sourcemaps are going into the dist folder, that why I'm getting the path this way. Even though, what is the path when deploying through Docker?

-  Sourcemaps are bundled and uploaded to sentry. Even though, when getting an issue report it doesn't show the source map but the bundled javascript file. Most likely related to the urlPrefix, but I couldn't figure it out, any idea?